meta-lmp-bsp/dynamic-layers/meta-ti-bsp/recipes-support/ti-mfgtool-files/ti-mfgtool-files_0.1.bb

```diff
@@ -36,5 +36,7 @@ do_deploy() {
 
 # Make sure the fitImage and u-boot are deployed
 do_deploy[depends] += "virtual/bootloader:do_deploy"
+# for tiboot3*.bin
+do_deploy[mcdepends] += "mc::k3r5:virtual/bootloader:do_deploy"
```

Comment:
Explain why in the commit message.

Reply:
We need  two U-Boot builds ( two different defconfigs are used from the same source tree) to be deployed: one for ARM64 core (`do_deploy[depends] += "virtual/bootloader:do_deploy`), another for R5 core (`do_deploy[mcdepends] += "mc::k3r5:virtual/bootloader:do_deploy`),

Reply:
I update the commit log